packages/web-react/docs/DocsSections.tsx

```diff
@@ -42,9 +44,14 @@ const DocsSection = (props: DocsSectionProps) => {
   );
 
   return (
-    <section {...styleProps} {...transferProps} className="UNSTABLE_Section">
-      {container === 'all' ? <Container>{content}</Container> : content}
-    </section>
+    <Section
+      size={hasPadding ? 'xsmall' : undefined}
+      {...styleProps}
+      {...transferProps}
+      hasContainer={container === 'all'}
```

Comment:
I'm wondering if the mixed order is correct. I think either destructured props, or our props should come first.